setup.go

```diff
@@ -94,6 +95,22 @@ func setup(c *caddy.Controller) error {
 			if len(args) == 1 {
 				fp.probeResistDomain = args[0]
 			}
+		case "serve_pac":
+			if len(args) > 1 {
+				return c.ArgErr()
+			}
+			if len(fp.pacFilePath) != 0 {
+				return errors.New("Parse error: serve_pac subdirective specified twice")
+			}
+			if len(args) == 1 {
+				fp.pacFilePath = args[0]
+				if !strings.HasPrefix(fp.pacFilePath, "/") {
+					fp.pacFilePath = "/" + fp.pacFilePath
+				}
+			} else {
+				fp.pacFilePath = "/proxy.pac"
+			}
+			fmt.Println("Proxy Auto-Config will be served at " + fp.hostname + fp.pacFilePath)
```

Comment:
Sure. I tried to use `github.com/mholt/caddy/caddyhttp/log` for it, but it doesn't export Print methods